testdata/evaluations.yml

```diff
@@ -0,0 +1,59 @@
+- expression: 'not x'
+  tests:
+    - variables: ['x']
+      result: false
+    - variables: ['y']
+      result: true
+- expression: 'x and y'
+  tests:
+    - variables: ['x', 'y']
+      result: true
+    - variables: ['x']
+      result: false
+    - variables: ['y']
+      result: false
+
+- expression: 'x or y'
+  tests:
+    - variables: []
+      result: false
+    - variables: ['x', 'y']
+      result: true
+    - variables: ['x']
+      result: true
+    - variables: ['y']
+      result: true
+- expression: 'x\(1\) or(y\(2\))'
+  tests:
+    - variables: ['x(1)']
+      result: true
+    - variables: ['y(2)']
+      result: true
+- expression: 'x\\ or(y\\\)) or(z\\)'
+  tests:
+    - variables: ['x\']
+      result: true
+    - variables: ['y\)']
+      result: true
+    - variables: ['z\']
+      result: true
+    - variables: ['x']
+      result: false
+    - variables: ['y)']
+      result: false
+    - variables: ['z']
+      result: false
+- expression: '\x or y\ or z\'
```

Comment:
Why does this expression not generate a syntax error? I'm reading these tokens in this line:

```yaml
- 'x'
- or
- 'y or'
- 'z'
```
I'm also reading an escaping character at the end of the string, meaning there's a missing escaped character. Are the semantics of an escaped "end of line" just an end of line? (so, ignore the backslash?)